<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EDE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../EstimulemaPY/imagenes/ema (1)_001.png</normaloff>../EstimulemaPY/imagenes/ema (1)_001.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="GB_Current_Control">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Current control CH1</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QSlider" name="Slider_CurrentCH1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_mACH1">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="suffix">
      <string> mA</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushBtnStop">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Polaridade</string>
    </property>
    <widget class="QRadioButton" name="rb_pos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Postivo</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_neg">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Negativo</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Selecionar canal</string>
    </property>
    <widget class="QRadioButton" name="rb_ch1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ch1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_ch2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ch2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_chx">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Chx</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_mACH1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_CurrentCH1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_CurrentCH1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
